docs/manuals/core/config/job-config.md

```diff
@@ -7,7 +7,7 @@ available in Taipy: the `standalone` mode and the `development` mode.
 With the _development_ mode, the jobs are synchronously executed one by one. The jobs are directly executed
 in the main thread at the submission. Note that with the _development_ mode, the submit method waits for the
 jobs to be finished before to return. Please refer to the
```

Comment:
```suggestion
jobs to be finished before returning.
```

Reply:
My suggestion is: 

For more information on how to submit jobs, refer to [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task).

Reply:
Haha yep I just saw it below! Sorry for the trouble! Have fixed and pushed the latest changes :D

Reply:
Ok cool, thanks! :)